src/main/java/org/jenkinsci/plugins/badge/ImageResolver.java

```diff
@@ -195,16 +195,18 @@ public StatusImage getTestResultImage(Integer testPass, Integer testTotal) {
      * @return
      */
     private String replaceTestResultSVG(String image, Integer testPass, Integer testTotal) {
+    	
+    	int passPercent = (testPass / testTotal) * 100;
```

Comment:
[FindBugs] WARNING: RCN: Nullcheck of testTotal at line 201 of value previously dereferenced in org.jenkinsci.plugins.badge.ImageResolver.replaceTestResultSVG(String, Integer, Integer)